active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,572 @@
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipButtonsSample.jpg" height="600"></a>
+
+### Bleeding content
+
+Edge to edge content can be added to a teaching tip by setting the BleedingContent proprerty. The location of bleeding content can be set to the top or bottom of a teaching tip by setting the BleedingContentPlacement property.
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            BleedingContent="Assets/Giraffe.png"
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipBleedingContentSample.jpg" height="600"></a>
+
+### Add an icon
+
+An icon can be added to the side of the title and subtitle using the IconSource property.
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            IconSource="SaveIcon.png"
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipIconSample.jpg" height="600"></a>
+
+### Enable light-dismiss
+
+A teaching tip may be light-dismiss enabled so that it will dismiss when a user scrolls or interacts with other elements of the application.
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            IsLightDismissEnabled="True"
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipLightDismissSample.jpg" height="600"></a>
+
+### Preferred placement
+
+Preferred placement of a teaching tip can be set using the TeachingTipPlacementMode property. If the preferred placement would not leave enough room for the teaching tip to show, another placement mode will automatically be chosen.
```

Comment:
Could you add a comment that this placement works exactly the same as [FlyoutPlacementMode](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Primitives.FlyoutPlacementMode)?

Reply:
```suggestion
Preferred placement of a teaching tip can be set using the TeachingTipPlacementMode property. Like FlyoutPlacementMode, if the preferred placement mode would not leave room for the teaching tip to show, another placement mode will be automatically chosen. 
```

Reply:
@MikeHillberg Does this read better?

Reply:
Calling out the Auto property is good.  I thought it would also be useful to say that everything else is the same in behavior as FlyoutPlacementMode.

Reply:
I have added your suggested change. Any thoughts on the phrasing?